desktop/contextMenu/index.js

```diff
@@ -0,0 +1,21 @@
+const { ContextMenuListener } = require('electron-spellchecker');
+
+// We are using a custom ContextMenuBuilder and SpellcheckHandler to become more flexible
+const ContextMenuBuilder = require('./ContextMenuBuilder');
+const SpellCheckHandler = require('./SpellCheckHandler');
+
+window.spellCheckHandler = new SpellCheckHandler(5);
+
+// Avoid an error on Windows and Linx when body is not yet available
+window.onload = () => {
+  window.spellCheckHandler.attachToInput();
+};
+window.spellCheckHandler.switchLanguage(navigator.language);
+
+let contextMenuBuilder = new ContextMenuBuilder(window.spellCheckHandler);
+// eslint-disable-next-line no-unused-vars
+let contextMenuListener = new ContextMenuListener(info => {
+  if (info.inputFieldType === 'none' && !info.isEditable) return;
```

Comment:
I think we may want to consider only showing the context menu for the editor div. I'm seeing spell check on the login, tag, search, etc inputs.

Reply:
This can be easily controlled with `spellchecker="false"` on the DOM element. I'm going to add it to the obvious fields as a demo.